content/reliable_tx.md

```diff
@@ -28,33 +28,37 @@ Applications face additional challenges, in the event of power or network loss,
+
+Each validated ledger has a canonical order in which transactions apply. This order is deterministic based on the final transaction set of the ledger. In contrast, each `rippled` server's in-progress ledger is calculated incrementally, as transactions are received. The order in which transactions execute provisionally is usually not the same as the order in which transactions execute to build a new validated ledger. This is one reason why the provisional outcome of a transaction may be different than the final result. For example, a payment may achieve a different final exchange rate depending on whether it executes before or after another payment that would consume the same offer.
 
 
 
 ### LastLedgerSequence
 
-[`LastLedgerSequence`](transactions.html#lastledgersequence) is an optional parameter of all transactions.  This instructs the Ripple network that a transaction must be validated on or before a specific ledger instance.  The transaction will never be included in a ledger instance with a higher sequence number.
+[`LastLedgerSequence`](transactions.html#lastledgersequence) is an optional parameter of all transactions.  This instructs the Ripple Consensus Ledger that a transaction must be validated on or before a specific ledger instance.  The Ripple Consensus Ledger never includes a transaction in a ledger instance whose sequence number is higher than the transaction's `LastLedgerSequence` parameter.
 
 Use the `LastLedgerSequence` parameter to prevent undesirable cases where a transaction is not promptly validated yet could become viable at some point in the future. Gateways and other back-end applications should specify the `LastLedgerSequence` parameter on every transaction. Automated processes should use a value of 4 greater than the sequence number of the last validated ledger[1] to ensure that a transaction is validated or rejected in a predictable and timely fashion.
 
-Applications using rippled APIs should explicitly specify a `LastLedgerSequence` when submitting transactions.  When using ripple-lib or Ripple-REST, a `LastLedgerSequence` is automatically included.  Applications using those APIs are recommended to use the `LastLedgerSequence` calculated by the API.
+Applications using rippled APIs should explicitly specify a `LastLedgerSequence` when submitting transactions. RippleAPI uses the `maxLedgerVersion` field of [Transaction Instructions](rippleapi.html#transaction-instructions) to specify the `LastLedgerSequence`. RippleAPI automatically provides an appropriate value by default. You can specify `maxLedgerVersion` as `null` to intentionally omit `LastLedgerSequence`, in case you want a transaction that can be executed after an unlimited amount of time.
```

Comment:
The consequences are as laid out in the opening paragraphs of the article -- 

> Applications which fail to implement best practices are at risk of the following errors:
> 
> 1. Submitting transactions which are inadvertently never executed.
> 2. Mistaking provisional transaction results for their final, immutable results.
> 3. Failing to find authoritative results of transactions previously applied to the ledger.

I think that's sufficient.